docs/userGuide/contentAuthoring.md

```diff
@@ -385,6 +385,10 @@ While authoring your website, you may want to have your own CSS or Javascript fi
 - Author your `<style>` elements for CSS files and `<link>` elements for Javascript files using HTML as shown below.
     - Ensure that your URLs start from the root directory, by using <code>{<span></span>{baseUrl}}/</code> when you are referencing your files.
 
+- Use `<head-top>` and `<head-bottom>` tags to control where your content will be placed.

```

Comment:
"to control where your content will be placed" may be quite vague, so we can be more explicit here.

`... tags to insert the content at the top and bottom of <head> respectively.`